config/locales/devise.en.yml

```diff
@@ -9,10 +9,10 @@ en:
     failure:
       already_authenticated: "You are already signed in."
       inactive: "Your account is not activated yet."
-      invalid: "Incorrect email or password. After 5 login attempts, accounts are locked for 30 minutes."
+      invalid: "Invalid email or password"
       locked: "Your account is locked."
       last_attempt: "You have one more attempt before your account is locked."
-      not_found_in_database: "Incorrect email or password. After 5 login attempts, accounts are locked for 30 minutes."
+      not_found_in_database: "Invalid email or password"
```

Comment:
Restoring these to their original value.

Reply:
I liked our customizations! :) Why reset them? If in support of the string compare below, well, OK, I guess that's life.

Reply:
Never mind -- I see our customizations live on below.

Strange to have the same key in both files. I guess the idea is that our definition shadows this definition. I'd prefer to only have the key in one file. Non-blocking.